### ISSUE TITLE
Investigate testing issues for fly.io

### ISSUE THREAD

**Issue by ehmatthes** [ehmatthes/django-simple-deploy#165]
Integration tests are failing regularly with fly.io, but if I leave the deployed site open it works when using it through a browser session. I think this is something to do with the lowest-level resources not always supporting rapid requests.

Sometimes even opening a Python session and using requests to issue a simple request to the home page fails, and then starts to work again a minute later. The smallest fly instance is pretty small; perhaps I should be using a slightly larger instance for testing?

- [x] Make sure to upgrade fly CLI if there's an update available. Fly is rapidly evolving, and sometimes issues go away after upgrading CLI. See note below. (It updates automatically now.)
- [x] Specify a better-resourced fly instance when running integration tests, and see if rate of passing tests improves. (This no longer seems to be an issue.)
- [ ] Consider asking Fly.io staff about this.

**Comment by ehmatthes** [comment]
Upgrading Fly CLI
---

Here's the warning when there's an update available:

```
Update available 0.0.418 -> v0.0.420.
Run "fly version update" to upgrade.
```

If this is the cause of numerous issues, we can report this to the user and bail until they upgrade. We don't want to run the upgrade, because that can be a slow process, ie initiating a brew update.

**Comment by ehmatthes** [comment]
This seems to no longer be an issue.
